submarineswaprpc/submarineswap_server.go

```diff
@@ -0,0 +1,125 @@
+//go:build submarineswaprpc
```

Comment:
This fie is a copy of LND sub server and definitely doesn't fit here.
We need a standalone grpc server that doesn't depend on LND.